### DISCUSSION TITLE
Cannot Load IWD2 Area Graphics Files

### DISCUSSION THREAD

**Discussion by LuminousWolf** [NearInfinityBrowser/NearInfinity#176] [Q&A] [answered]
Hi everyone! I just started using NearInfinity. It looks really promising, but I'm having some trouble getting started.

NearInfinity won't load any area graphics files for IWD2. Trying to open TIS, WED, or bitmap files triggers an error message ("Error while loading" for TIS, "error reading" for the others). When I click on a MOS file, nothing at all happens. The ARE loads fine, however.

DLTCEP behaves similarly. It loads the ARE but can't read any of the other files.

It's a new installation of IWD2 patched to v2.01, and the game plays just fine.

I also looked at the source code and found that NearInfinity is unable to identify the TIS files as palette or PVRZ. I tried to look at the hex code for the TIS files, but I can't find them anywhere in the IWD2 installation, not even in the BIF archives. I'm guessing they're encrypted or compressed. That's as far as I got.

Any help would be appreciated!

**Comment by Argent77** [comment]
Could you open the debug console (menu `Tools` > `Show Debug Console`) and copy the whole error message that is printed after you tried to open a TIS or MOS file?

**Reply by LuminousWolf** [reply]
[nidebuglog.txt](https://github.com/NearInfinityBrowser/NearInfinity/files/14791536/nidebuglog.txt)

Sure, here's the output when I try to open AR1000.TIS.

**Comment by Argent77** [comment]
Thanks. It looks like NearInfinity can't resolve or access the game path for some reason and is therefore unable to locate the game resources.

Is the game installed in a specially protected folder (e.g. `C:\Program Files` or `C:\Program Files (x86)`) or on a remote drive with special permissions? Are you able to open the game installation directory in the Windows Explorer or any files in the game folder (e.g. icewind2.ini with a text editor)?

Does it help if you run NearInfinity as administrator?

**Reply by LuminousWolf** [reply]
It is installed in Program Files (x86). I can open the game directory and files in Explorer just fine, but running as administrator doesn't help.

**Comment by Argent77** [comment, marked as answer]
I can only think of two causes for this behavior:
1. You haven't made a full installation of the game, so some data has to be fetched from the CD. I haven't really tested that scenario, but it should theoretically work if you make the CD available in the CD drive. Otherwise, reinstall the game and make sure to install everything to the harddrive. (The full installation should have installed about 1.4 GB of data on disk.)
2. The current location (`C:\Program Files (x86)`) causes trouble for external tools to access game files. In that case you could relocate the game folder to a less restricted location and try again.

To test the second option,
- Create a new folder on a safe location, e.g. `C:\Games`
- Copy the whole `Icewind Dale II` game folder to the new location
- In the new location open the file `icewind2.ini` with a text editor. It can be found inside the game folder. Scroll down to the \[Alias\] section and adapt the paths of the "HD0" and "CDx" entries to match the new location.

**Reply by LuminousWolf** [reply]
Thanks a lot, that worked! I uninstalled it and did a full installation in a different folder. Now I can open all the files.
